neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -46,19 +46,19 @@ private bool CheckValidators(ApplicationEngine engine)
         internal override bool Initialize(ApplicationEngine engine)
         {
             if (!base.Initialize(engine)) return false;
-            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MaxBlockSize), new StorageItem
+            engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_MaxBlockSize), () => new StorageItem
```

Comment:
Why change to `GetAndChange()`?

Reply:
the original code relied on the overwriting behaviour of the old `Add()` implementation.  This now causes unit test failures because the value already exists and an exception is thrown. By using `GetAndChange` we basically reproduce the old `Add()` behaviour. 

Alternatively I can insert some code like the following in all the affected tests to clear the backend data.
```C#
foreach (KeyValuePair<StorageKey, StorageItem> pair in snapshot.Storages.Find())
{
   snapshot.Storages.DeleteInternal(pair.Key);
}
```

Reply:
I prefer to fix the unit test